.travis.yml

```diff
@@ -6,15 +6,17 @@ cache:
   - $HOME/.cache/pip
   - .arca/cache
 install:
-- pip install -r test_requirements.txt
-- pip install 'arca[docker]>=0.2'
+- python -m pip install pipenv --upgrade
+- pipenv install --deploy --dev
+- pipenv install --ignore-pipfile 'arca[docker]>=0.2'
```

Comment:
When you run this, it (confusingly, even when ``ignore-pipfile`` is set) modifies the Pipfile. When the repository is modified, cache of content fragments from courses from the base repository is disabled. (This is done so local changes are visible straight away and the lessons are not rendered from cache).

Maybe we should add 'arca[docker]>=0.2' to the --dev dependencies? An average user will not be installing those anyway and it will make things easier...

Reply:
The `--dev` dependencies are needed to run tests. As `naucse` becomes more about working  on the codebase (and the content moves to separate repos), more and more of average users *will* want to install them.

Despite that, I don't have a problem adding `arca[docker]` as a dev dependency.

Reply:
Looks like adding`arca[docker]` to dev deps while `arca` is already in regular deps is not something pipenv was prepared for: https://github.com/pypa/pipenv/issues/2330

Reply:
Adding `arca[docker]` to regular deps.